03-transactions.md

```diff
@@ -766,54 +766,54 @@ Upon successful exchange of `tx_complete` messages, each peer must at
 least cover a minimum estimated fee. Here is how to calculate the minimum
 fee for each participant.
 
-The minimum witness weight for an input is 110.
+The minimum witness weight for an input is 107.
 
 In the following, the initiator has provided one input (P2WPKH), one change output
 (P2WPKH), and the funding output. The contributor has provided two
 inputs (P2WPKH) and two change outputs (P2WPKH).
 
 Assuming a `feerate` of 253 per kiloweight, the initiator's minimum fee is
-calculated as follows.  Note that the feerate is rounded down to the nearest satoshi.
+calculated as follows.  Note that the feerate is rounded up to the nearest satoshi.
```

Comment:
Since `110` was a semi-arbitrary number above the minimum, it was ok to round down. Now, since we're using the minimum weight, always round up (to avoid being beneath the desired rate).